### ISSUE TITLE
Not able to install yearfrac (version = 0.4.4)

### ISSUE THREAD

**Issue by ashwinassysh** [kmedian/yearfrac#27]
Hey 

I am trying to install yearfrac version 0.4.4. But i am getting below error:

```
Collecting yearfrac==0.4.4
 Downloading yearfrac
 Preparing metadata (setup.py): started
 Preparing metadata (setup.py): finished with status 'error'
 error: subprocess-exited-with-error
  
 × python setup.py egg_info did not run successfully.
 │ exit code: 1
 ╰─> [3 lines of output]
   error in yearfrac setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after version specifier)
     numpy>=1.14.*
        ~~~~~~^
   [end of output]
  
 note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> yearfrac

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

I am not specifying numpy version in my requirements file. Not sure whats going wrong here.

**Comment by ulf1** [comment]
@ashwinassysh What python version are you using?

**Comment by ulf1** [comment]
@ashwinassysh have you tried to install the latest version `yearfrac==0.4.8`

**Comment by ashwinassysh** [comment]
@ulf1 I am on python version 3.8. There were no issues in creating this environment with 0.4.4 version. Not sure why suddenly it stopped working.

**Comment by Sousuke-Cheng** [comment]
@ashwinassysh  i'm also facing the same problem. Have you solved it?
I am using xlcalculator 0.4.2 and it requires yearfrac 0.4.4 to be installed.
Upon installing it, the same thing as the above popped up. 
I used it in a docker which used Python 3.8, with the pip version being the newest. 
Perhaps it's 3.8.16, but I'm not sure about that.
However, my colleague is able to install it with the Python version being 3.8.5 and the pip version being 23.1.

**Comment by ulf1** [comment]
@Sousuke-Cheng @ashwinassysh You are referring to a problem that have been fixed in `yearfrac==0.4.8`. Please use `xlcalculator>=0.5.0` which uses the fixed version `yearfrac==0.4.8`
